src/admin-orderable-add/orderable-program-add.html

```diff
@@ -0,0 +1,45 @@
+<div class="modal" role="dialog" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h1 ng-show="!vm.canEdit">{{'adminOrderableProgram.titlePreviewing' | message: {programName: vm.programsMap[vm.programOrderable.programId].name} }}</h1>
+                <h1 ng-show="!vm.editMode">{{'adminOrderableProgram.titleAddProgram' | message}}</h1>
```

Comment:
As in previous html files. One time binding will be better. But only for the translations that won't change. Like this one